benchmarks/src/main/java/com/yahoo/omid/benchmarks/utils/TimeValueTimeUnit.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2011-2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.omid.benchmarks.utils;
+
+import java.util.concurrent.TimeUnit;
+
+// TODO: Maybe this should be moved to the commons module as a general utility
+public class TimeValueTimeUnit {
```

Comment:
When we specified the metrics configuration both in client and server side (e.g. default-omid.yml) we specify the output Freq in a similar way. I tried to be consistent with that. Also in the configuration, for parameters such as the run length of the experiment, this way you can easily specify periods like 1 HOUR, 1 DAY, or 25 MINUTES